Exercise 3/Exercise3.rst

```diff
@@ -1,7 +1,15 @@
-Look at the file titled titled "Match This Output.png" in this directory.
+Basics of Cooking
+#################
+Making Pizza
+************
+To make pizza you'll need to: 
```

Comment:
You'll need to have a blank line before the start of any bulleted or numeric list